specs/daito/Sushi_Fileshare_Design.md

```diff
@@ -0,0 +1,95 @@
+# Sushi Fileshare Design
+
```

Comment:
Although the focus of this design is to provide file share support.  Please also include how it works for volumes so that other people can understand why they need to be separate.